src/internal/FluenceClientImpl.ts

```diff
@@ -73,10 +73,11 @@ export class FluenceClientImpl extends FluenceClientBase implements FluenceClien
         script = wrapFetchCall(script, callBackId, resultArgNames);
         const particle = await build(this.selfPeerIdFull, script, data, ttl, callBackId);
 
-        return new Promise<T>((resolve, reject) => {
+        const prFetch = new Promise<T>(async (resolve, reject) => {
             this.fetchParticles.set(callBackId, { resolve, reject });
```

Comment:
I guess we can drop this fetch line completely. Public fetch api works differently (see api.ts file)

Reply:
That's just a little bit of legacy left after FOSDEM preparations